dotnet/src/webdriver/ErrorResponse.cs

```diff
@@ -42,58 +39,45 @@ public ErrorResponse()
         /// </summary>
         /// <param name="responseValue">A <see cref="Dictionary{K, V}"/> containing names and values of
         /// the properties of this <see cref="ErrorResponse"/>.</param>
-        public ErrorResponse(Dictionary<string, object> responseValue)
+        public ErrorResponse(Dictionary<string, object?>? responseValue)
         {
             if (responseValue != null)
             {
-                if (responseValue.ContainsKey("message"))
+                if (responseValue.TryGetValue("message", out object? messageObj)
+                    && messageObj?.ToString() is string message)
                 {
-                    if (responseValue["message"] != null)
-                    {
-                        this.message = responseValue["message"].ToString();
-                    }
-                    else
-                    {
-                        this.message = "The error did not contain a message.";
-                    }
+                    this.Message = message;
                 }
-
-                if (responseValue.ContainsKey("screen") && responseValue["screen"] != null)
+                else
                 {
-                    this.screenshot = responseValue["screen"].ToString();
+                    this.Message = "The error did not contain a message.";
```

Comment:
If it `didn't contain a message`, then let's review the specification.

Reply:
It's worth noting that the current logic doesn't actually work for the stack trace property, because it is returned as a string instead of an `object[]`. It looks like there's something to fix here, with the spec I can give it a shot.

![image](https://github.com/user-attachments/assets/f4e260ec-4ebf-493b-96a5-a81f75076858)

Reply:
I am just reading changes and see you introduced `The error did not contain a message.` magic message. Probably I missed something.

Reply:
The diffs are much easier to look at with whitespace disabled https://github.com/SeleniumHQ/selenium/pull/14672/files?w=1

Reply:
https://www.w3.org/TR/webdriver2/#errors - seems should be `stacktrace`, not `stackTrace`

Reply:
I don't mind we can handle `stacktrace` || `stackTrace`, but it should be `string`. Interesting, somebody did it especially, probably we can see it in git history why...

Reply:
In any case, in this PR let's keep things as simpler as possible, and in the following PR I would propose to align this with the specification. When `Screenshot` can be removed. And look at other bindings, how they handle `error` object.